README.md

```diff
@@ -37,20 +37,27 @@ bugs and rough edges.
 
 Python 3.8 is recommended.
 Python 3.7 through 3.10 are supported and tested.
+Make sure to have a development version of python installed locally as well.
 
-TorchDynamo requires the latest development/nightly build of PyTorch or
-building [PyTorch from source]. TorchDynamo also requires [functorch]
-from the PyTorch repository.  The Makefile target `make setup_nightly_gpu`
-contain the commands used by our CI to setup dependencies.
+To use GPU back ends (and in particular Triton), please make sure to have cuda
+11.7 installed locally.
```

Comment:
11.6 also should work.  Not sure about other versions.

Reply:
Since I listed the binaries for 11.7 and don't want people to get into trouble with the cuda toolchain not matching the install I left 11.7 here.
Do you want me to move this into the "binary" install and mention in the from source that the user should have 11.6+ locally?

Reply:
Updated the PR to reflect this.

Reply:
It might be good to drop a link for other versions.